mlir/lib/Dialect/Affine/Transforms/LoopTiling.cpp

```diff
@@ -173,6 +173,12 @@ void LoopTiling::getTileSizes(ArrayRef<AffineForOp> band,
 }
 
 void LoopTiling::runOnOperation() {
+  if (cacheSizeInKiB <= 0) {
+    getOperation().emitError(
```

Comment:
I think use `mlir::emitError` is better, and `cacheSizeInKiB` is uint64_t, it's not less than zero. Maybe you can change it to:
```
if (cacheSizeInKiB == 0) {
  mlir::emitError(mlir::UnknownLoc::get(&Pass::getContext()),
                  "cache size cannot be 0");
}
```
And please add a test. You can refer to #68762.

Reply:
Thank you for the pointers! Changes made to in the new commit.